R2Northstar/NorthstarLauncher
# Remove uses of Autohook from `loghooks.cpp`

### Code review:
- The original function is prefixed with `o_p` so any times where the old AUTOHOOK was calling the original function it should now be calling that instead.
- The hook function is prefixed with `h_` so any times where we would be wanting to call the hooked function from other functions should now be calling that instead

### Testing:
- Check logs to make sure that all of the changed hooks are being created properly
- Make sure that the various logging methods work as intended. (server spew only logs when `spewlog_enable 1` is set)
